app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -186,6 +193,42 @@ const SurveyObservations = () => {
     </IconButton>
   );
 
+  const groupType = {
+    mandatory: 'Mandatory fields have not been filled out in your file',
+    unsupported_header: 'Column headers in your file are not supported',
+    miscellaneous: 'Miscellaneous'
```

Comment:
should this say something along the lines of:
```suggestion
    miscellaneous: 'Miscellaneous errors in your file'
```

Reply:
ok.  Just FYI though ... the miscellaneous is to bucket things that are not assigned a grouping yet.  Not sure if we'll keep this concept longterm .. but for now it's helpful

Reply:
cool cool